mne/viz/decoding.py

```diff
@@ -58,15 +62,21 @@ def plot_gat_matrix(gat, title=None, vmin=0., vmax=1., tlim=None,
     if ax is None:
         fig, ax = plt.subplots(1, 1)
 
+    # Define color limits
+    if vmin is None:
+        vmin = np.min(gat.scores_)
+    if vmax is None:
+        vmax = np.max(gat.scores_)
```

Comment:
silly question, isn't that the matplotlib default? If so this should be unnecessary.